### ISSUE TITLE
fman 1.17 fails to start in Manjaro with GLIBC_2.28 error

### ISSUE THREAD

**Issue by IgorKrupenja** [fman-community/fman#471]
With #468 fixed, I get a new error when trying to start 1.17:

```
$ /opt/fman/fman                                                     [16:49:11]
[2201] Error loading Python lib '/opt/fman/libpython3.5m.so.1.0': dlopen: /usr/lib/libc.so.6: version `GLIBC_2.28' not found (required by /opt/fman/libpython3.5m.so.1.0)
```

Not sure but looks like fman was compiled against glibc version 2.28 but the latest in Manjaro repos is 2.27.

**Comment by IgorKrupenja** [comment]
Can confirm that this is fixed in 1.18 and I can successfully launch fman.

**Comment by mherrmann** [comment]
Fantastic! Thanks for letting me know.
